osu.Game/Online/Chat/StandAloneChatDisplay.cs

```diff
@@ -89,9 +93,10 @@ private void postMessage(TextBox sender, bool newtext)
 
             if (text[0] == '/')
                 ChannelManager?.PostCommand(text.Substring(1), Channel.Value);
-            else
+            else {
+                text = ChannelManager?.ProcessInlineCommands(text);
```

Comment:
why is this calling a second call to channel manager when the next line is a call to channel manager